src/test/java/com/imgix/test/TestSrcSet.java

```diff
@@ -135,10 +133,10 @@ public void testNoParametersSignsUrls() {
         for (String srcLine : srcsetSplit) {
 
             src = srcLine.split(" ")[0];
-            assert(src.contains("s="));
-            generatedSignature = src.substring(src.indexOf("s=")+2);
+            assertThat(src, containsString("s="));
```

Comment:
More sonar-lint related changes. Using `assertThat` open the door to more meaningful test error messages, since it has an optional error string that can be passed in. Moreover, the test is now more explicit about what type of return value it's expecting.

This change was necessary in several places.